src/app/concern/file-uploader/file-uploader.component.ts

```diff
@@ -38,31 +46,46 @@ export class FileUploaderComponent implements OnInit {
     this.dropArea.nativeElement.classList.remove("highlight");
   }
 
-  public handleDrop(e: any): void {
-    // const dt = e.dataTransfer;
-    // this.uploadedFiles = dt.files;
-    this.upload();
+  public handleDrop($event: DragEvent): void {
+    const itemList: DataTransferItem[] = Array.from($event.dataTransfer.items);
+    const filesAllowed: string[] = this.acceptedFileTypes.split(",");
```

Comment:
Can't we declare the allowedFileTypes as array instead of splitting into an array?

Reply:
good point, makes sense. i will do this now

Reply:
pr updated @babul3012